sourcecode/hub/resources/views/components/layout.blade.php

```diff
@@ -57,7 +58,7 @@ class="btn-close m-2 align-self-start"
         @endif
 
         <div @class(['container-md' => !$expand, 'flex-grow-1'])>
-            <main class="h-100 d-flex flex-column">
+            <main @class(['h-100', 'd-flex', 'flex-column', 'main-container-grid' => $layout === 'grid', 'main-container-list' => $layout === 'list'])>
```

Comment:
It would be better if the layout component took a generic `main_classes` property, rather than letting details specific to sublayouts leak into it.